src/storage/src/source/mysql/snapshot.rs

```diff
@@ -404,7 +405,18 @@ pub(crate) fn render<G: Scope<Timestamp = GtidPartition>>(
 
                 let mut snapshot_staged = 0;
                 for (table, outputs) in &reader_snapshot_table_info {
-                    let query = format!("SELECT * FROM {}", table);
+                    // MySQL supports invisible columns, which must be requested explicitly in the projection
+                    // to be retrieved.
+                    let columns = outputs
+                        .iter()
+                        .flat_map(|info| {
+                            info.desc
+                                .columns
+                                .iter()
+                                .map(|col_info| format!("`{}`", col_info.name))
+                        })
+                        .join(", ");
```

Comment:
Hm, I think this might be doing a different thing than what we need. Here you iterate over all outputs and concatenate the name of the columns from each one of them. So if I understand correctly, if the same table is ingested twice in Materialize we will end up with a query like `SELECT col1, col2, col1, col2` and transfer twice the amount of data. Am I reading this right?